### ISSUE TITLE
 Update outdated dependencies (but only after vite bundling is merged in mirador core)

### ISSUE THREAD

**Issue by lutzhelm** [ProjectMirador/mirador-image-tools#54]
`npm install` with the [mirador4-compat-clean branch](https://github.com/ProjectMirador/mirador-image-tools/tree/mirador4-compat-clean) lists several dependencies that are no longer maintained. Some of those dependencies are transitive dependencies via mirador, so breaking down the necessary details should be postponed until ProjectMirador/mirador#3985 is merged (which should already get rid some of those warnings).
